packages/vertexai/src/models/generative-model.ts

```diff
@@ -66,9 +67,15 @@ export class GenerativeModel {
     requestOptions?: RequestOptions
   ) {
     if (!vertexAI.app?.options?.apiKey) {
-      throw ERROR_FACTORY.create(VertexError.NO_API_KEY);
+      throw new VertexAIError(
+        VertexAIErrorCode.NO_API_KEY,
+        'Missing Firebase app API key'
```

Comment:
Restore the detailed message as mentioned above, and the one below as well.

Reply:
Don't know why I changed these in the first place 🤔 Done